Deploy/Systemd/ss-susi-server.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Starting SUSI Server for SUSI Linux
+After=multi-user.target
+
+[Service]
+Type=oneshot
+ExecStart=/home/pi/SUSI.AI/susi_linux/susi_server/susi_server/bin/restart.sh
```

Comment:
same

Reply:
The idea of "restarting" a program by providing a systemd service is silly.

Systemd already has command `systemctl restart some-service` to restart a service which it manages. So you have to either:
- Let `systemd` starts _susi\_server_. Then if you want to restart _susi\_server_, just use `systemctl restart`.
- If there is no way to let _susi\_server_ managed by `systemd`, just restart directly by `restart.sh` script. No need to make systemd service (delete this file).

Reply:
@hongquan , when we try to boot SUSI server on raspi (after a shutdown) , there are many times when the port 4000 is still occupied from the previous boot. To start again we have to use the restart.sh script instead of start.sh .
And we try to restart the service instead, it won't empty out the port while the server is also not running

Reply:
> the port 4000 is still occupied from the previous boot

Silly. How come there is a process which can live across boot session to occupy port 4000.

It is just that some service has started on reboot, earlier than your service and occupy that port.

The proper fix is to figure out which service take over that port, and prevent it from starting, not to have a "restart" service.